src/Table/TableCell.js

```diff
@@ -4,24 +4,22 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import type { ElementType, Node } from 'react';
 import classNames from 'classnames';
+import deprecated from 'prop-types-extra/lib/deprecated';
```

Comment:
Please, let's avoid the depreciation. Introduction breaking changes save maintainer times.